### PR TITLE
Test Unicode code point escapes with more than 6 hex digits

### PR DESCRIPTION
**Pull request by mathiasbynens**

### CODE REVIEW

File: es6.js [hunk 1]
```diff
@@ -47,8 +47,9 @@ bom:for(;;)break﻿bom;
 // line terminators
 lineFeed:0
 0;
-carriageReturn:00;
-carriageReturnLineFeed:0
+carriageReturn:0
+0;
+carriageReturnLineFeed:0
```
Comment (michaelficarra) [inline es6.js]: Sneaking this in here? Please submit that as a separate PR.

Comment (mathiasbynens) [inline es6.js]: Oh, sorry, this was unintentional. My editor normalized the line endings automatically.

Comment (mathiasbynens) [inline es6.js]: Let’s add an empty `.editorconfig` file to prevent this from happening again? Reduces friction for contributors. #11

**Comment by mathiasbynens** [comment]
Closing in favor of #12.
